src/main/java/com/ncc/aif/ValidateAIFCli.java

```diff
@@ -0,0 +1,597 @@
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.riot.RDFDataMgr;
+import org.apache.jena.riot.RDFFormat;
+import org.topbraid.jenax.statistics.ExecStatistics;
+import org.topbraid.jenax.statistics.ExecStatisticsListener;
+import org.topbraid.jenax.statistics.ExecStatisticsManager;
+import org.topbraid.shacl.vocabulary.SH;
+import picocli.CommandLine;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Spec;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.concurrent.Callable;
+
+/**
+ * A command-line AIF validator.  For details, see <a href="https://github.com/NextCenturyCorporation/AIDA-Interchange-Format">the AIF README</a>
+ * section entitled, <i>The AIF Validator</i>.
+ */
+@CommandLine.Command(name = "validateAIF",
+        sortOptions = false,
+        synopsisHeading = "%nUsage: ",
+        descriptionHeading = "%nDescription: ",
+        optionListHeading = "%nOptions:%n",
+        description = "Used to validate Turtle files with extension .ttl",
+        mixinStandardHelpOptions = true,
+        version = "1.1.0")
+public class ValidateAIFCli implements Callable<Integer> {
+
+    // Program return codes from the AIF Validator.
+    public enum ReturnCode {
+        SUCCESS, VALIDATION_ERROR, USAGE_ERROR, FILE_ERROR
+    }
+
+    //=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+    // Package Constants
+    // ----------------------------
+    // Error strings
+    static final String ERR_MISSING_ONT_FLAG = "Must use one of these flags: --ldc | --program | --ont";
+    static final String ERR_TOO_MANY_ONT_FLAGS = "Can only use one of these flags: --ldc | --program | --ont";
+    static final String ERR_MISSING_FILE_FLAG = "Must use one of these flags: -f | -d";
+    static final String ERR_TOO_MANY_FILE_FLAGS = "Can only use one of these flags: -f | -d";
+    static final String ERR_SMALLER_THAN_MIN = "%s must be at least %d";
+    // Logging strings
+    static final String START_MSG = "AIF Validator";
+
+    //=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+    // Internal Constants
+    // ----------------------------
+    // Logger
+    private static final Logger logger = (Logger) (org.slf4j.LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME));
+    // ViolationThreshold
+    private static final String ABORT_PARAMETER_STRING = "Abort parameter";
+    private static final int DEFAULT_MAX_VIOLATIONS = -1;
+    private static final int MINIMUM_MAX_VIOLATIONS = 3;
+    // Profiling
+    private static final int LONG_QUERY_THRESH = 2000;
+    // Threading
+    private static final String THREAD_COUNT_STRING = "Thread count";
+    private static final int MINIMUM_THREAD_COUNT = 1;
+
+    //=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+    // Command Line Arguments
+    // ----------------------------
```

Comment:
Some formatting things in Usage output:
- Previously, option descriptions were designed to fit 80-char screens.  If it's possible, it would be nice to preserve this behavior.  There are line breaks being added which are making it look bad in 80 characters.
- One dash options (`-v`) are indented differently from two-dash switches (`--abort`).  Any idea why?

One dash options (-v) are indented differently from two-dash switches (--abort).

Reply:
Fixed